lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1718,7 +1729,7 @@ def api_client
         creds = GRPC::Core::CallCredentials.new(authentication.updater_proc)
         creds = ssl_creds.compose(creds)
         @client = Google::Logging::V2::LoggingServiceV2::Stub.new(
-          'logging.googleapis.com', creds)
+          @logging_api_url, creds)
```

Comment:
Technically, in the case of an RPC stub, it's a _host_ (a standalone host name), not a _URL_ (`protocol://host:port/path`). In fact, we would need to reconcile the need to specify the host on the gRPC path and the full URL on the non-gRPC path.

Reply:
Maybe we can let them always specify the full URL like `https://logging.googleapis.com`. Then for the gRPC path, we throw away the `protocal` part?

WDYT?

Reply:
That could work. Will gRPC take a `host:port` spec? Also, what happens if the URL has a path? We should probably specify what the expected format is.